packages/web-components/fast-foundation/src/avatar/avatar.template.ts

```diff
@@ -25,6 +25,7 @@ export const avatarTemplate: (
             part="link"
             href="${x => (x.link ? x.link : void 0)}"
             style="${x => (x.color ? `color: var(--avatar-color-${x.color});` : void 0)}"
+            ${ref("control")}
```

Comment:
I'm wondering if this should be `link` to match with the "part". I have a proposal hopefully coming this week regarding adding refs and aligning to the named parts.

Reply:
I agree, but it looks like there is an attr that is already called `link`
![image](https://user-images.githubusercontent.com/37851220/135124637-ce8e52c1-b8fd-49fb-b9c9-580b0a9e0c54.png)